nginx/certificates.sls

```diff
@@ -1,20 +1,20 @@
-{% from 'nginx/map.jinja' import nginx with context %}
+{%- set tplroot = tpldir.split('/')[0] %}
+{%- from tplroot ~ '/map.jinja' import nginx, pillar_namespace with context %}
 
 include:
-  - nginx.service
+  - {{ tplroot }}.service
```

Comment:
I think most formulas use the following syntax
```
include:
  - .service
```
and I think it should work the same.